### DISCUSSION TITLE
[notification] ElNotification is not displayed

### DISCUSSION THREAD

**Discussion by Jhonatan-1504** [element-plus/element-plus#9208] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `notification`

## Description

I've tried to use the ElNotification component, but it doesn't work, previously it worked in its version of 2.1.8, now I'm using version 2.2.6, my vue is 3.2.37

### Additional comments

_(empty)_

**Comment by zeshhaan** [comment]
I have the same question. Is there a way to control the styles of this notification component?
